cmd/kube-proxy/app/server.go

```diff
@@ -99,12 +99,6 @@ type Options struct {
 	CleanupIPVS bool
 	// config is the proxy server's configuration object.
 	config *kubeproxyconfig.KubeProxyConfiguration
-
-	// The fields below here are placeholders for flags that can't be directly mapped into
```

Comment:
Why removing this? Have the deprecated flags already been removed?

I don't think this is right.

/cc @ncdc PTAL.

Reply:
Correct, we can't do this yet